argoverse/evaluation/detection/utils.py

```diff
@@ -157,9 +157,10 @@ def accumulate(
         logger.info(f"{dt_filtered.shape[0]} detections")
         logger.info(f"{gt_filtered.shape[0]} ground truth")
         if dt_filtered.shape[0] > 0:
-            ranked_detections, scores = rank(dt_filtered)
-            metrics = assign(ranked_detections, gt_filtered, cfg)
-            cls_to_accum[class_name] = np.hstack((metrics, scores))
+            ranked_dts, ranked_scores = rank(dt_filtered)
```

Comment:
[nit] `detections` is more readable than `dts`. Looks like we changed to latter in multiple places.

Reply:
If we do go this route, I think we should switch all of the variables over to the more verbose naming (e.g., `dt_filtered` -> `detections_filtered`). What do you think?

Reply:
how about we keep it generally as it was before? I think `dt_filtered` is actually simpler to read than `detections_filtered`. That variable name seems to be getting too long